code/citadel/dogborgstuff.dm

```diff
@@ -356,7 +356,15 @@
 		/obj/item/documents/syndicate,
 		/obj/item/disk/nuclear,
 		/obj/item/bombcore,
-		/obj/item/grenade)
+		/obj/item/grenade,
+		/obj/item/melee,
+		/obj/item/key,
+		/obj/item/kitchen/knife,
+		/obj/item/storage,
+		/obj/item/restraints,
+		/obj/item/clockwork,
+		/obj/item/device/pda,
+		/obj/item/device/assembly/flash)
```

Comment:
be Janidog, be unable to do anything to defend yourself from anything if hacked.

seems legit.